book/src/kimchi/lookup.md

```diff
@@ -125,7 +126,10 @@ $$
 
 Since we would like to allow multiple table lookups per row, we define multiple **queries**, where each query is associated with a **lookup selector**.
 
-At the moment of this writing, the `ChaCha` gates all perform $4$ queries in a row. Thus, $4$ is trivially the largest number of queries that happen in a row.
+Previously, ChaCha20 was implemented in Kimchi but has been removed as unneeded.
```

Comment:
suggestion: s/as unneeded/as it has become unneeded.

Reply:
Fixed.